File: python/1.py
class Solution(object):
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        length = len(nums);
        r = [];
        for m in range(0,length):
            for n in range(m + 1,length):
                if nums[m] + nums[n] == target:
                    r.append(m);
                    r.append(n);
        return r;

File: C++/1.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) 
    {
        int len = nums.capacity();
        vector<int> r;
        for(int i = 0;i < len;++i)
        {
            for(int j = i + 1;j < len;++j)
            {
                if(nums.at(i) + nums.at(j) == target)
                {
                    r.push_back(i);
                    r.push_back(j);
                }
            }
        }
        return r;
    }
};